README.md

```diff
@@ -44,18 +60,29 @@ lb_public_url = http://xxx.xxx.xxx.xxx
 
 > The application is being deployed to the compute instances asynchronously, and it may take a couple of minutes for the URL to serve the application.
 
-## Questions
+### MuShop Basic Topology
 
-If you have an issue or a question, please take a look at our [FAQs](./deploy/basic/FAQs.md) or [open an issue](https://github.com/oracle/oci-quickstart-cloudnative/issues/new).
+The following diagram shows the topology created by this stack.
+
+![MuShop Basic Infra](./images/basic/00-Free-Tier.png)
+
+## Getting Started with MuShop Complete
```

Comment:
MuShop Basic Topology and then Getting Stated with MuShop Complete?

Reply:
We should have a separator between the two

Reply:
done.